ui/desktop/app/templates/scopes/scope/projects/targets/target.hbs

```diff
@@ -48,6 +48,25 @@
         </Hds::ApplicationState>
       </bc.Body>
       <bc.Sidebar>
+        <div class='details-sidebar aliases' data-test-aliases-sidebar>
```

Comment:
none blocking comment: Are we using this data-test? I would suggest to delete it if we are not using it.

Reply:
ah good catch, I intended to use this for testing, will update

Reply:
removed!